src/lib/routes/admin-api/environments.ts

```diff
@@ -89,8 +98,11 @@ export class EnvironmentsController extends Controller {
                 openApiService.validPath({
                     tags: ['Environments'],
                     operationId: 'getProjectEnvironments',
+                    description:
+                        'Gets the environments that are available for this project',
```

Comment:
Something like this, maybe? What makes an environment available?

```suggestion
                    summary: 'Get environments available to a project',
                    description:
                        'Gets all environments that are available for the specified project. An environment is considered available if ... what?',
```